packages/ui/src/components/va-select/VaSelect.vue

```diff
@@ -498,18 +489,13 @@ export default defineComponent({
       }
     }
 
-    // Focus and keyboard navigation
-
-    const isFocusedComputed = computed(() => {
-      // If we show dropdown content that means select is focused
-      return isFocused.value || showDropdownContent.value
-    })
+    const openDropdown = () => {
+      if (props.disabled || props.readonly) { return }
```

Comment:
This condition can be moved to `showDropdown` method.

Reply:
fixed it